data/language/en-GB.txt

```diff
@@ -3695,6 +3695,10 @@ STR_6589    :Show window buttons on the left
 STR_6590    :Show the window buttons (e.g. to close the window) on the left of the title bar instead of on the right.
 STR_6591    :Staff member is currently fixing a ride and can’t be fired.
 STR_6592    :Staff member is currently inspecting a ride and can’t be fired.
+STR_6593    :LSM Launched Roller Coaster
+STR_6594    :A low to the ground roller coaster propelled by Liniar Synchronous Motors.
+STR_6595    :Multi-Looping Roller Coaster
+STR_6596    :A steel rollercoaster that is known for making multipe inversions through it's lay-out.
```

Comment:
A steel-tracked roller coaster that features a wide variety of twisting inversions throughout its layout

Reply:
Sounds good! Thanks for the recommendation.

Reply:
> Sounds good! Thanks for the recommendation.

I was going to try and get the word "disorientating" in there, but getting these descriptions right is hard. I often leave it till last.